README.md

```diff
@@ -1,109 +1,88 @@
 2. Run ProtonUp-Qt and select Steam
 3. Click `Add Version`, select Luxtorpeda and press `Install`
 4. Restart Steam
 5. In Steam game properties window select "Force the use of a specific Steam Play
    compatibility tool" and select "Luxtorpeda".
 <img height="220px" src="https://user-images.githubusercontent.com/54072917/139227152-0536ac68-0d4b-44bf-be88-42105f5c3dd6.png" />
 
-## Installation (debug build from source)
+### 🎮 On Steam Deck
 
-### Prerequisites
-You will need Rust, Cargo, OpenSSL, Godot4 and export templates.
+There's two easy ways to install Luxtorpeda on Steam Deck. You can either install it from Desktop Mode or if you're using [Decky Loader](https://decky.xyz/) you can install it using the [Wine Cellar](https://github.com/FlashyReese/decky-wine-cellar) plugin.
 
-#### 1. Install required software
+#### From Desktop mode
+1. From the STEAM menu, select Power, then Switch to Desktop.
+2. Open the 'Discover Store'.
+3. Search and download 'ProtonUp-Qt'.
+4. Follow the same steps as [installation On Linux](#-on-linux) above.
+5. On the desktop, there's a link to 'Return to Gaming Mode'. Select that to get back to standard Steam Deck UI. You can also go through the system menus to Log Off to get back.
 
-**Official method**:
-- [Rust and Cargo](https://www.rust-lang.org/learn/get-started): You can install Rust and Cargo using rustup.
-- [Godot 4](https://godotengine.org/download/linux/): Download and install Godot 4 and export templates from their official webpage.
-- OpenSSL: Install OpenSSL from your distribution's package manager.
-
-**Terminal installation** (for those who prefer the terminal):
+#### From Decky Loader
+1. Download and [install Decky Loader]([https://decky.xyz/)](https://github.com/SteamDeckHomebrew/decky-loader?tab=readme-ov-file#-installation).
+2. Download the 'Wine Cellar' plugin using [these instructions](https://github.com/SteamDeckHomebrew/decky-loader?tab=readme-ov-file#-getting-started).
+3. Go to the Decky Loader tab on the Quick Access menu and find the Wine Cellar plugin.
+4. Oress "Manage" to get taken to a new screen:
+5. Choose 'Luxtorpeda' from the menu and install it.
+6. Restart Steam
+7. In Steam game properties window select "Force the use of a specific Steam Play compatibility tool" and select "Luxtorpeda".
 
-- **Debian, Ubuntu, and variants**: Open a terminal and run:
+For further installation methods, see our [Other installation methods](#other-installation-methods) section.
 
-    ```bash
-    $ sudo apt install cargo libssl-dev godot
-    ```
+## 💬 Join the Community
+Need help? have questions? want to chat about fun old games? We are a welcoming community to all. 
 
-- **Fedora**: Use the following command:
+**[Discord channel](https://discord.gg/8mFhUPX)**: welcome to our Discord channel
 
-    ```bash
-    $ sudo dnf install openssl-devel rust cargo godot
-    ```
+**[Steam Group](https://steamcommunity.com/groups/luxtorpedaplay)**: join our Steam group.
```

Comment:
Same as above, would prefer to keep communication in the discord, and make mention that bugs and requests should be put in github.